frontend/encore/typescript.rst

```diff
@@ -34,6 +34,20 @@ also configure the `ts-loader options`_ via a callback:
 If React assets are enabled (``.enableReactPreset()``), any ``.tsx`` file will be
 processed as well by ``ts-loader``.
 
+
+Here is a sample ``tsconfig.json`` file :
+
+.. code-block:: javascript 
+
+{
+    "compilerOptions": {
+        "jsx": "react" // If you want to use react 
+    },
+    "include": [
+        "./assets/js/**/*"
+    ]
+}
```

Comment:
Is this something that Encore should configure for you? I don’t know typescript we’ll - what does the include do?

Reply:
Hi, 
Well, Typescript (as far as I understand it) actually compiles your files to JS before webpack handles/bundles them. This options tells the compiler which files it should compile or not. By default, the compiler will not do anything at all as the `files` array is empty by default. 

This simply tells the compiles, it should handle all files (having the .ts extension) in the `assets/js` folder.